### DISCUSSION TITLE
FSF is soliciting proposals for a process for identifying candidates and appointing board members

### DISCUSSION THREAD

**Discussion by gorshunovr** [rms-support-letter/rms-support-letter.github.io#5791] [General]
The board of directors is soliciting proposals from qualified consultants to assist in creating a transparent, formal process for identifying candidates and appointing board members who are wise, capable, and committed to the FSF's mission. The FSF intends to rewrite the by-laws in a way that binds the organization to transparency in its choice of directors. This process will establish ways for FSF associate members and supporters to meaningfully contribute to the discussion. The board is looking for proposals to be received by **Friday, April 2, 2021**.

Please email [info@fsf.org](mailto:info@fsf.org) with the subject **"Director Transparency Engagement"** for details.

[link](https://www.fsf.org/news/update-on-work-to-improve-governance-at-the-fsf)

**Comment by nukeop** [comment]
I volunteer!

**Comment by Aspie96** [comment]
I asked for details and got no answer

I am interested to know in why the president resigns
